frontend/src/static/js/v_authorship.js

```diff
@@ -169,7 +195,11 @@ window.vAuthorship = {
       let totalBlankLineCount = 0;
 
       files.forEach((file) => {
-        const lineCnt = file.authorContributionMap[this.info.author];
+        const contributionMap = file.authorContributionMap;
+        const lineCnt = this.info.isMergeGroup
+          ? Object.entries(contributionMap).reduce((acc, [author, cnt]) => (author !== '-' ? acc + cnt : acc), 0)
```

Comment:
Could consider refactoring this to another function, even though it is just a 1 liner, since this is not too easy to digest. Maybe call it `sumContributionsFromAllAuthors`.

Reply:
Yep refactored